test/integration/partner_category_filter_test.rb

```diff
@@ -2,8 +2,8 @@
 
 require 'test_helper'
 
-class PartnerCategoryFilterTest < ActionDispatch::IntegrationTest
-  setup do
+class PartnerCategoryFilterIntegrationTest < ActionDispatch::IntegrationTest
+  def setup
```

Comment:
I'm not sure which is correct but in test/controllers/concerns/partner_category_filter_test.rb we use:

`setup do`

Would be good to stick with one pattern.